profiles/api/webhook.py

```diff
@@ -19,9 +22,12 @@ def create_blueprint(profiles: Profiles, orcid_config: Dict[str, str],
 
     @blueprint.route('/orcid-webhook/<payload>', methods=['POST'])
     def _update(payload: str) -> Response:
+        LOGGER.info('POST request made to /orcid-webhook/%s', payload)
+
         try:
             orcid = uri_signer.loads(payload)
```

Comment:
can we log anything about `orcid` here with `info` level? `payload` is unreadable probably

Reply:
I added the `loads_unsafe` for logging purposes. This should translate the hashed payload into an ORCID without raising an exception.